WordPress/Docs/PHP/DiscouragedPHPFunctionsStandard.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Discouraged PHP Functions"
+    >
+    <standard>
+    <![CDATA[
+Use JSON instead of serialized data, which has known vulnerability problems with object injection.
```

Comment:
Here and in all other `<standard>` blocks, the description should be indented four spaces.
```suggestion
    Use JSON instead of serialized data, which has known vulnerability problems with object injection.
```